### DISCUSSION TITLE
using the script with gz image

### DISCUSSION THREAD

**Discussion by zacny** [Drewsif/PiShrink#265] [Show and tell]
Hello. Is it possible to use this script with compressed version of an image. Let say, I have large capacity disc, but it is almost empty. When I do image in that way:
`sudo dd if=/dev/sdb | pv | gzip > backup.img.gz`
I get an image that takes up practically as much space as the data inside it.
Is it possible to shrink `backup.img.gz` file, end up with `shrink-backup.img` that is almost the same size as `backup.img.gz`.
Can I do this in a single command?

**Comment by Drewsif** [comment]
You can not, PiShrink uses other tools that do not support working directly from a compressed image so there is no way to do it currently.
